client/src/components/InternshipCompanyInfo.jsx

```diff
@@ -1,57 +1,85 @@
-import { Avatar, Box, Typography } from '@mui/material';
+import { Box, Typography } from '@mui/material';
 import React from 'react';
 
+import amazonLogo from '../assets/company_logos/amazon_logo.svg';
+import appleLogo from '../assets/company_logos/apple_logo.svg';
+import genericLogo from '../assets/company_logos/generic.svg';
+import googleLogo from '../assets/company_logos/google_logo.svg';
+import metaLogo from '../assets/company_logos/meta_logo.svg';
+import microsoftLogo from '../assets/company_logos/microsoft_logo.svg';
+import netflixLogo from '../assets/company_logos/netflix_logo.svg';
+
 const InternshipCompanyInfo = (props) => {
+  const getCompanyIcon = (company) => {
+    let companyLowerCase = company.toLowerCase();
+    switch (companyLowerCase) {
+      case 'amazon':
+        return amazonLogo;
+      case 'apple':
+        return appleLogo;
+      case 'google':
+        return googleLogo;
+      case 'meta':
+        return metaLogo;
+      case 'microsoft':
+        return microsoftLogo;
+      case 'netflix':
+        return netflixLogo;
+      default:
+        return genericLogo;
+    }
+  };
+
   return (
-    <>
+    <Box
+      display="flex"
+      maxWidth={false}
+      sx={{
+        maxWidth: 'fit-content',
+      }}
+    >
       <Box
```

Comment:
nit: imo I think it might be better to use `<Avatar />` rather than `<Box />` because it has quite a few attributes such as `variant="rounded"`, `src`, and `alt` that are more specific to images (it is also easier for those who are reading your code to understand that this is an image rather than a regular box)